packages/manager/src/components/application/ApplicationIndex.tsx

```diff
@@ -13,10 +16,26 @@ const styles = StyleSheet.create({
   },
 });
 
-export default () => {
+export default withRouter(({ history }) => {
   const location = useLocation();
   const { applicationName } = useParams();
+  const [environment] = useState<IEnvironment>(config.environments[0]);
```

Comment:
I think the `environments[0]` should not be assume as the default. it should put `null` into here
And I also didn't found any code pass to the delete function. are you miss it ?

Reply:
@kunyan The environment is passed to the deleteApplication method call on line 25.